### PR TITLE
Establishes a script to remove user progress associated with a non-existent script.

### PR DESCRIPTION
**Pull request by ashercodeorg**
There are currently seven script IDs (5, 8, 20, 27, 29, 31, 37) for which production user progress exists without an associated script. This user progress (in particular the `user_scripts`) causes the associated users to not be able to load studio.code.org as the result of errors.

The JSON files with the data will be moved to AWS s3.

### CODE REVIEW

File: bin/remove_script_remnants [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      where(user_id: user_level['user_id']).
+      to_a.
+      map(&:serializable_hash)
+    data[:activities].flatten!
+  end
+
+  data
+end
+
+# Deletes all the data in data from the DB.
```
Comment (breville) [inline bin/remove_script_remnants]: data in data? :)

**Comment by breville** [comment]
obviously scary to be actually deleting the data (as opposed to just marking it as somehow unused) but this lgtm.

**Comment by ashercodeorg** [comment]
No doubt, ewjordan. Thus the second commit (the AR objects aren't too useful). And my intent is to dry-run the script (by not invoking the delete) until I'm confident of it.

File: bin/remove_script_remnants (hunk 1 above)
Comment (ashercodeorg) [reply bin/remove_script_remnants]: Fair enough, I'll reword to "all the data from the DB in data".

**Pull request merged**